interpreter/cling/docs/ReleaseNotes.md

```diff
@@ -8,7 +8,7 @@ describe the status of Cling in some detail, including major
 improvements from the previous release and new feature work.
 
 Note that if you are reading this file from a git checkout or the main
-[Cling web page](https://rawgit.com/root-project/cling/master/www/index.html),
+[Cling web page](https://root.cern/cling/),
```

Comment:
My comment was for this line.

Reply:
Oh. Should I change to the Github repo instead? Rawgit is discontinued and will shtit down at some point. Or how should I amend the commit message?

Reply:
We can host the static webpage of cling via github nowadays. Probably that's the best way to go here.

Reply:
So shall I point instead to https://github.com/root-project/cling/tree/master/www or to a future root-project.github.io/cling ?

Reply:
I've split the PR to facilitate.